snap/validate.go

```diff
@@ -300,7 +300,15 @@ func validateSocketAddrNetPort(fieldName string, port string) error {
 	return nil
 }
 
-func validateDescription(descr string) error {
+func ValidateSummary(summary string) error {
+	// TODO is 1024 a good value? what is allowed by the store?
```

Comment:
from my checking it seems review-tools accepts 128 and snapcraft accepts only 78. In the store itself this seems configurable and we would have to ask what it is configured to? anyway 1024 is probably too much